### PR TITLE
DP-25131: [Chapter 55 report] Fix broken links and update content

### PR DESCRIPTION
**Pull request by DmitriiTartynovLCM**
Changes:
* Links and content were updated as requested.
* Added the announcement to the top of the page.
* Specified the latest NodeJS version in the `package.json` for Volta. Later versions fail to build the `node-sass` binary.
* Committed the `package-lock.json` file for future consistency. Lack of this file also caused an update of assembled/minified CSS/JS without any actual changes in code, probably because installed versions were different from the previously used ones.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,5 +24,8 @@
     "pump": "^1.0.1",
     "run-sequence": "^1.2.2",
     "uglify-js": "^2.7.3"
+  },
+  "volta": {
```
Comment (clairesunstudio) [inline package.json:28]: in the other projects, we are using `nvm` to manage node versions. Is volta serving the same purpose?

Comment (clairesunstudio) [reply package.json:28]: It failed when I ran `npm i` directly. 
Screen Shot 2022-06-13 at 2 59 10 PM

I still had to use `nvm` to switch to node 10, then it installed successfully for me.

Can we just add a .nvmrc file to the project instead?

**Comment by clairesunstudio** [comment]
I added a PR to update the basic setup here 
https://github.com/massgov/chapter55/pull/8

**Comment by clairesunstudio** [comment]
other than the comment I had above, changes look good to me! 

please look at my linked PR and let me know if you have questions. Once we got that in and have this one updated, I will merge this into dev and will be auto-deployed to the Github pages link, and I will send it to DPH to review before we publish it. thanks!

File: package.json (hunk 1 above)
Comment (DmitriiTartynovLCM) [reply package.json:28]: We could keep both `.nvmrc` file and `volta` section in the `package.json`, if you don't mind.

**Comment by clairesunstudio** [comment]
You can add Volta to README.md and squash and merge the PR. Thanks!

**Pull request merged**